draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4303,26 +4307,36 @@ better still.
 
 <section title="Changes from RFC 5245">
 
-<t> Following is the list of changes from RFC 5245
-<list style="symbols">
-<t> The specification was generalized to be more usable with any
-protocol and the parts that are specific to SIP and SDP were moved to
-a SIP/SDP usage document <xref
-target="I-D.ietf-mmusic-ice-sip-sdp"/>. </t>
-<t> Default candidates, multiple components, ICE mismatch detection,
-subsequent offer/answer, and role conflict resolution were made
-optional since they are not needed with every protocol using ICE. </t>
-<t>
-With IPv6, the precedence rules of RFC 6724 are used instead of the
-obsoleted RFC 3483 and using address preferences provided by the host
-operating system is recommended. </t>
-<t> Candidate gathering rules regarding loopback addresses and IPv6
-addresses were clarified. </t>
-
+<t> The purpose of this specification is to:
+<list>
+<t>Clarify procedures in RFC 5245.</t>
+<t>Make technical changes, due to bugs in RFC 5245 and based
+on feedback from the community that has implemented and
+deployed ICE applications based on RFC 5245.</t>
+<t>Make the procedures signaling protocol independent, by
+removing the SIP and SDP procedures. Procedures specific
+to a signaling protocol will be defined in separate usage
+documents. <xref target="I-D.ietf-mmusic-ice-sip-sdp"/> defines
+the ICE usage with SIP and SDP.</t>
 </list>
-
 </t>
 
+<t> The following technical changes have been done:
+<list style="symbols">
+<t>Aggressive nomination removed.</t>
+<t>The procedures for calculating candidate pair states
+and scheduling connectivity checks modified.</t>
+<t>Check list set definition added.</t>
```

Comment:
The definition updates probably don't belong to technical changes

Reply:
I'll remove the check list set definition.